docs/jwst/references_general/apcorr_reffile.inc

```diff
@@ -262,7 +262,7 @@ For IFU data the APCORR reference files are in ASDF format. The aperture correct
 varies with wavelength and the contents of the files are shown below. The radius,
 aperture correction and error are all 2D arrays. Currently the 2nd dimension does
 not add information, but in the future it could be used to provide different aperture
-corrections for different radii. 
+corrections for different radii.
 
 :Data model: `~jwst.datamodels.MirMrsApcorrModel`
```

Comment:
Here and elsewhere, these are not clickable.  Why did you choose to only fix the NRSIFU model and no others here?

Reply:
Hmm I was just going through the doc build log. If the log does not complain, I won't notice. I can double check to see why this isn't caught.

Reply:
Maybe it is in the log but I just wasn't look at the right place. Anyways, I went ahead and attempt to fix all similar occurrences in this file.